pipelines/build/devkit/make_devkit.sh

```diff
@@ -43,9 +43,25 @@ patch -p1 < ../Tools.gmk.patch
 
 devkit_target="${ARCH}-linux-gnu"
 
-# Perform devkit build
-cd make/devkit && make TARGETS=${devkit_target} BASE_OS=${BASE_OS} BASE_OS_VERSION=${BASE_OS_VERSION}
+if [ "${BASE_OS}" = "rhel" ]; then
```

Comment:
should tha tbe a ==?

Reply:
No. `=` is the POSIX standard however bash does allow `==` too.